src/ui/screens/settings/ViewSubscriptionManagement/SubscriptionManagementItem.qml

```diff
@@ -82,7 +119,7 @@ ColumnLayout {
         spacing: VPNTheme.theme.listSpacing
 
         Layout.alignment: Qt.AlignVCenter
-        Layout.bottomMargin: VPNTheme.theme.listSpacing
+        Layout.bottomMargin:  VPNTheme.theme.listSpacing
```

Comment:
Nit: An unneeded space sneaked in here.

Reply:
Those sneaky spaces are always up to no good